src/model/transformations/responses.js

```diff
@@ -19,17 +19,34 @@ export function stateToModel(
   const responsesModel = collectedVariablesFinal.map(cv =>
     Response.stateToRemote({ ...state, collectedVariable: cv }),
   );
-  const mappingModel = responsesModel.map(r => {
-    const { x, y } = collectedVariablesStore[r.CollectedVariableReference];
 
+  let attributeModel = [];
+
+  const mappingModel = responsesModel.map(r => {
+    const { x, y, isCollected } = collectedVariablesStore[r.CollectedVariableReference];
     // Table : Fix lines and look into columns
     const MappingTarget =
       type === QUESTION_TYPE_ENUM.MULTIPLE_CHOICE ? `${x}` : `${x} ${y}`;
+    
+      if (isCollected === false ) {
+        attributeModel.push({
+          "AttributeValue": "NoDataByDefinition",
+          "AttributeTarget": MappingTarget
+        },)
+      }
     return { MappingSource: r.id, MappingTarget };
   });
 
-  return {
-    Response: responsesModel,
-    Mapping: mappingModel,
+  if (type == 'TABLE') {
```

Comment:
I think we have a const somewhere for this string.

Reply:
Yes, we have. I have replaced with it now.